backend/index.js

```diff
@@ -0,0 +1,73 @@
+const express = require("express");
+const app = express();
+const port = 3000;
+const bodyParser = require("body-parser");
+const fs = require('fs');
+const cors = require('cors')
+
+app.use(cors())
+
+app.use(bodyParser.urlencoded({ extended: false }))
+
+app.use(bodyParser.json())
+
+app.use(bodyParser.urlencoded({ extended: true }));
+
+
+let tasks = []
+app.get("/task/getAll", (req, res) => {
+    res.json(tasks.map(x => x));
```

Comment:
why do we need this tasks.map( x => x) ?
It looks like we can just return the array of tasks:
`res.json(tasks)`